### PR TITLE
Av1 and av2 common library usage

### PR DESCRIPTION
**Pull request by piekczyk**
# [Av1 and av2 common library usage](https://app.shortcut.com/oazo-apps/story/7589/av1-av2-common-library-usage)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- replaced `decodeTriggerData` with `decodeTriggerDataAsJson` from common library
  
## How to test 🧪
  <Please explain how to test your changes>

- try to add, remove and update maker and aave triggers (correct numbers should be displayed)

### CODE REVIEW

File: features/automation/common/state/autoBSTriggerData.ts [hunk 1]
```diff
@@ -62,7 +63,7 @@ function mapAutoBSTriggerData(autoBSTriggers: { triggerId: number; result: Resul
       maxBuyOrMinSellPrice: new BigNumber(maxBuyOrMinSellPrice.toString()).isEqualTo(maxUint256)
         ? maxUint256
         : new BigNumber(maxBuyOrMinSellPrice.toString()).div(new BigNumber(10).pow(18)),
-      continuous,
+      continuous: continous,
```
Comment (piotrkonowrocki) [inline features/automation/common/state/autoBSTriggerData.ts:66]: There should be just
```
continuous,
```
but one of them is with typo, why? Is this some external data that we don't control?

File: features/automation/optimization/autoTakeProfit/tests/autoTakeProfitTriggerData.test.ts [hunk 2]
```diff
@@ -70,18 +69,23 @@ describe('autoTakeProfit Form Change tests', () => {
       const triggers = [
         {
           triggerId: idOne,
-          result: ['', 7, new BigNumber(1), new BigNumber(4)],
+          result: {
```
Comment (piotrkonowrocki) [inline features/automation/optimization/autoTakeProfit/tests/autoTakeProfitTriggerData.test.ts:72]: Could those number be saved in some common consts? What are they?

File: package.json [hunk 3]
```diff
@@ -49,7 +49,7 @@
     "@myetherwallet/mewconnect-connector": "^0.1.8",
     "@next/bundle-analyzer": "^12.0.4",
     "@next/mdx": "^12.0.4",
-    "@oasisdex/automation": "1.2.7-alpha.2",
+    "@oasisdex/automation": "1.2.8-alpha.1",
```
Comment (piotrkonowrocki) [inline package.json:52]: Is that supposed to be commited? Alpha version?

Comment (piekczyk) [reply package.json:52]: nope, we will still need to update it

File: features/automation/optimization/autoTakeProfit/tests/autoTakeProfitTriggerData.test.ts (hunk 2 above)
Comment (piekczyk) [reply features/automation/optimization/autoTakeProfit/tests/autoTakeProfitTriggerData.test.ts:72]: these are just random mocks in test, not worth to save them as consts I guess

File: features/automation/common/state/autoBSTriggerData.ts (hunk 1 above)
Comment (piekczyk) [reply features/automation/common/state/autoBSTriggerData.ts:66]: yep, I believe I mentioned it on daily and Adam said that this is how it looks like in one of our datasources or sth

**Pull request merged**